### DISCUSSION TITLE
More integration of theme generator and colors page

### DISCUSSION THREAD

**Discussion by EvgeniiaVak** [saadeghi/daisyui#2356] [Ideas / feature requests]
A use case: I want to quickly prototype theme colors right in the https://daisyui.com/theme-generator/ but then I also want to quickly discover what variables to use in the code using this page https://daisyui.com/docs/colors/ (here I would like to be able to see the colors I created in the theme generator page). 

Also theme generator page lacks some colors. Maybe it could be a separate even if super long page with both theme generator and colors specs. Another cool thing would be to be able to share it via copy/pasting the url.

**Comment by saadeghi** [comment]
I completely agree with these ideas.  
Currently the theme generator only shows the required color names, to demonstrate how easy it is to define the colors (optional colors like `primary-content`, etc will be generated automatically)  
But of course devs can define values for optional colors and other CSS variables as well, or even [add custom styles](https://daisyui.com/docs/themes/#-8) for a theme

About other ideas you suggested, I would work on them when I get free time, but currently I'm stuck with fixing issues and that has a higher priority.
